MdeModulePkg/Test/Library/MockBaseResetSystemLibNull/MockBaseResetSystemLibNull.c

```diff
@@ -0,0 +1,37 @@
+/** @file

```

Comment:
nit: naming convention-wise, I would leave Null out of the name, it is a bit confusing (what is a Mock Null Lib?). These are mocks of the actual libs, so I would do Mock<LibraryClass>, i.e. MockResetSystemLib, etc.

Reply:
Why should the "Base" be left out of the name?

Reply:
Base is referring to a specific instance of the library class (the base class), the mock really doesn't have anything to do with a particular instance of the library class, it just mocks that library class. If a given library class needed multiple mock libs, that is feasible, but the others would just be called MockSpecialVersionLibraryClass. That being said, I don't have a big preference with Base being in there or not as there is the above use case where the lib might be mocked more than once, my comment was more about Null.